pdfminer/pdffont.py

```diff
@@ -654,6 +655,17 @@ def __init__(self, rsrcmgr, spec, strict=settings.STRICT):
             if strict:
                 raise PDFFontError('Encoding is unspecified')
             name = 'unknown'
+        if type(name) is PDFStream:
+            if 'CMapName' in name:
+                name = name.get('CMapName').name
```

Comment:
It would be great if we could have a more meaningful vairable name here. Something like `CMapName` would also do provided it makes sense ( since the variable `name`, in this context does mean something else ).